guide/contribute/formatting.md

```diff
@@ -0,0 +1,400 @@
+---
+layout: guide
+title: Formatting
+description: Visual examples of formatting options available for content authors.
+nav_order: 11
+parent: Contribute
+permalink: /guide/contribute/formatting/
+image: /assets/images/guide/contribute/formatting-preview.jpg
+---
+
+# Formatting
+
+This page showcases the various formatting and layout options available for content. This allows editors to better understand their toolbox and access reference code. It also allows for designers to see the design system in one place. The design source file is a public Figma community file you can find [here](https://www.figma.com/community/file/862622015964353400/Bitcoin-Designers-site). To improve the design, please start with the Figma file and make a proposal in Slack or Github before implementing.
+
+
+## Basic markdown formatting
+
+We use [Markdown](https://daringfireball.net/projects/markdown/) when writing content. Markdown is a plain-text formatting syntax that helps us better prepare our text for the web. Below you can find an overview of commonly used syntax elements.
+
+### Text formatting
+
+Text can be **bold**, _italic_, or ~~strikethrough~~.
+
+```markdown
+**bold**
+_italic_
+~~strikethrough~~
+```
+There should be whitespace between paragraphs.
+
+### Headers
+
+```markdown
+# h1
+## h2
+### h3
+#### h4
+##### h5
+###### h6
+```
+
+# h1 header
+## h2 header
+### h3 header
+#### H4 header
+##### h5 header
+###### h6 header
+
+### Blockquote
+
+```markdown
+> This is a blockquote text.
+```
+
+> This is a blockquote text.
+
+
+### Links
+
+### External Links
+
+```markdown
+[Link to another page](https://bitcoin.org/bitcoin.pdf).
+```
+
+[Link to another page](https://bitcoin.org/bitcoin.pdf).
+
+#### Internal links
+
+```markdown
+[Linking]({{ '/guide/getting-started/why-bitcoin-is-unique/' | relative_url }})
+```
+[Linking]({{ '/guide/getting-started/why-bitcoin-is-unique/' | relative_url }})
+
+### Images
+
+Let's start with a very wide image that extends beyond the content with on desktops. Note how a different images is shown on mobile. This can be used to reformat image content to a portrait format.
+
+```html
```

Comment:
The liquid tags between this code block gets compiled by Jekyll and shows the html for `<figure><img />` instead of the code to demonstrate.

To resolve this wrap the code blocks with `{{% raw %}}` and `{{% endraw %}}`

See https://whatapalaver.co.uk/liquid-codeblocks-jekyll-markdown

Reply:
Are you sure? It [shows](https://deploy-preview-109--sad-borg-390916.netlify.app/guide/contribute/formatting/#images) the code that should be embedded and the image preview of how it looks once it's there.

Reply:
@johnsBeharry Tried with wrapping blocks with liquid and raw, but got this error when building
```
   Jekyll Feed: Generating feed for posts
    Liquid Warning: Liquid syntax error (line 68): Unexpected character % in "{{% raw %}}" in guide/contribute/formatting.md
    Liquid Warning: Liquid syntax error (line 81): Unexpected character % in "{{% endraw %}}" in guide/contribute/formatting.md
    Liquid Warning: Liquid syntax error (line 96): Unexpected character % in "{{% raw %}}" in guide/contribute/formatting.md
    Liquid Warning: Liquid syntax error (line 108): Unexpected character % in "{{% endraw %}}" in guide/contribute/formatting.md
    Liquid Warning: Liquid syntax error (line 123): Unexpected character % in "{{% raw %}}" in guide/contribute/formatting.md
    Liquid Warning: Liquid syntax error (line 136): Unexpected character % in "{{% endraw %}}" in guide/contribute/formatting.md
    Liquid Warning: Liquid syntax error (line 157): Unexpected character % in "{{% raw %}}" in guide/contribute/formatting.md
    Liquid Warning: Liquid syntax error (line 174): Unexpected character % in "{{% endraw %}}" in guide/contribute/formatting.md
```

Reply:
Regardless, I've pushed a new commit so you can test it out, I've no idea how to address this further.

Reply:
@pavlenex damn sorry i've added double curly braces it should be 1 pair.

{% raw %}
```liquid
{% picture...
%}
```
{% endraw %}

Reply:
Ha! The guy in that doc used double curly braces! This is what we get for trying to fix stuff at 1AM! Works now, thanks for checking it this late <3